src/cogent3/util/table.py

```diff
@@ -1940,6 +1941,14 @@ def tolist(self, columns=None):
 
         return result
 
+    @c3warn.deprecated_callable(version="2023.9", reason="PEP8", new="to_list")
```

Comment:
The version is the released version by which this will be removed. Change it to 3 months from now, so the version when `Table.tolist()` will disappear is "2024.1"